### PR TITLE
Allow for setting custom controlling process

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: test/ex_webrtc/peer_connection_test.exs [hunk 1]
```diff
@@ -199,6 +199,19 @@ defmodule ExWebRTC.PeerConnectionTest do
 
   # API TESTS
 
+  test "controlling process" do
```
Comment (mickel8) [inline test/ex_webrtc/peer_connection_test.exs:202]: Not sure if we want this test. I feel like it's pretty ugly :thinking:

Comment (LVala) [reply test/ex_webrtc/peer_connection_test.exs:202]: I mean, if you have written it already...

**Review by LVala — APPROVED** [review]
SGTM, although do we have any particular need for this feature?

**Comment by mickel8** [comment]
> SGTM, although do we have any particular need for this feature?

Yup, it's needed to allow for spawning PeerConnection under supervision tree

**Pull request merged**